docs/kyma/docs/04-06-use-helm.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Use Helm
```

Comment:
I'm missing use case, and even a reference on how and what I can install with Helm